pyfftw/pyfftw.pxd

```diff
@@ -373,6 +373,48 @@ cdef extern from 'fftw3.h':
 
     double FFTW_NO_TIMELIMIT
 
+    ctypedef enum fftw_r2r_kind:
```

Comment:
Three times exactly the same `enum`. Why? Is it really needed? Couldn't we use everywhere `fftw_r2r_kind`?

Reply:
I am not familiar with cython; I tried aliases `fftwf_r2r_kind = fftw_r2r_kind`, `ctypedef fftw_r2r_kind fftwf_r2r_kind` etc, but they do not work, so I naively copy the enum.

I am not sure if `fftw_r2r_kind` can be safely used for `float` and `long double`.

Reply:
Alright, let's go like that. At least, it's simple and it works.